cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs

```diff
@@ -176,6 +176,7 @@ impl frame_system::Config for Runtime {
 	type Version = Version;
 	type AccountData = pallet_balances::AccountData<Balance>;
 	type SystemWeightInfo = weights::frame_system::WeightInfo<Runtime>;
+	type SystemExtensionsWeightInfo = weights::frame_system_extensions::WeightInfo<Runtime>;
```

Comment:
DQ but why are they called `SystemExtensions` and not `TransactionExtensions`?

Reply:
It's not really `SystemExtensions`, it's more like `System Extensions WeightInfo`, if that makes sense. Could have been `SystemTransactionExtensionsWeightInfo` but it felt too long.

Reply:
`SystemWeightInfo` is already a misnomer since we normally use `WeightInfo` so maybe `ExtrnsionsWeightInfo` is enough?

Reply:
Renamed all instances to `SystemWeightInfo`.